### ISSUE TITLE
Application Error - Before Create and renaming 

### ISSUE THREAD

**Issue by henricop** [getinsomnia/insomnia#7028]
### Expected Behavior

I would like to move between folders, create new ones and rename them

The error happens frequently since the last insomnia update! 

### Actual Behavior

```
TypeError: Cannot read properties of null (reading 'index')
    at file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-f6dc2321.js:119:69992
    at jh (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-f6dc2321.js:44:24296)
    at F6 (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-f6dc2321.js:44:42448)
    at d_ (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-f6dc2321.js:44:36651)
    at Vs (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-f6dc2321.js:42:3288)
    at dge (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-f6dc2321.js:44:41325)
    at dc (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-f6dc2321.js:44:40309)
    at YO (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-f6dc2321.js:44:35758)
    at N (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-f6dc2321.js:29:1614)
    at Immediate.he [as _onImmediate] (file:///Applications/Insomnia.app/Contents/Resources/app.asar/index-f6dc2321.js:29:1985)
    at process.processImmediate (node:internal/timers:476:21)
```

### Reproduction Steps

- Go to any collection
- Access a directory that already contains created requests
- Create a new request and try to rename it by double clicking on the name
- Erase the value completely
- Enter the new value

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Additional Information

![Screen Shot 2024-01-26 at 08 51 24](https://github.com/Kong/insomnia/assets/54844857/003bfe7c-3f44-4b13-884a-dabda0d2ee90)

### Insomnia Version

8.5.1

### What operating system are you using?

macOS

### Operating System Version

macOS Big Sur - Version 11.6.2 (20G314)

### Installation method

https://insomnia.rest/download

### Last Known Working Insomnia version

-

**Comment by gatzjames** [comment]
Hi @henricop can you try the latest Insomnia 8.6.0 and let me know if it still happens?

- I can't reproduce the issue even on 8.5.1 so I'm thinking it could be an error from a plugin. Can you disable your plugins and see if that's the case?
- Also any logs you could share when this error happens would be great help to figure this issue. You can open the DevTools by opening `View > Toggle DevTools` and find any errors that get logged to the Console.

**Comment by henricop** [comment]
It was resolved in version: 8.6.0 !! Thank you @gatzjames
